tasks/components.py

```diff
@@ -41,97 +41,142 @@ def has_type_component(content):
     return any(l.startswith('type Component interface') for l in content)
 
 
-def check_component(file, content):
+# // TODO: (components)
+# The migration of these components is in progresss.
+# Please do not add a new component to this list.
+components_to_migrate = [
+    "comp/aggregator/demultiplexer/component.go",
+    "comp/core/config/component.go",
+    "comp/core/flare/component.go",
+    "comp/core/telemetry/component.go",
+    "comp/dogstatsd/replay/component.go",
+    "comp/dogstatsd/server/component.go",
+    "comp/forwarder/defaultforwarder/component.go",
+    "comp/logs/agent/component.go",
+    "comp/metadata/inventoryagent/component.go",
+    "comp/netflow/config/component.go",
+    "comp/netflow/server/component.go",
+    "comp/otelcol/collector/component.go",
+    "comp/remote-config/rcclient/component.go",
+    "comp/trace/agent/component.go",
+    "comp/trace/config/component.go",
+    "comp/process/apiserver/component.go",
+    "comp/core/workloadmeta/component.go",
+]
+
+# // TODO: (components)
+# The migration of these components is in progresss.
+# Please do not add a new component to this list.
+components_missing_implementation_folder = [
+    "comp/dogstatsd/statsd",
+    "comp/core/tagger",
+    "comp/forwarder/orchestrator/orchestratorinterface",
+    "comp/core/hostname/hostnameinterface",
```

Comment:
These are weird cases since this were created to fulfil the Otel project requirements and they are a bit of a weird state